Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.py

```diff
@@ -0,0 +1,1346 @@
+            "Source IP": activity.get("sourceip", ''),
+            "Source Port": activity.get("sourceport", ''),
+            "Destination Port": activity.get("destinationport", ''),
+            "Categories": ", ".join(activity_data.get('category', [])),
+            "Type": activity.get("type", ''),
+            "Action": activity.get("verdict", ''),
+            "Date & Time": activity_data.get('timestamp_string')
+        }
+        activity_list.append(new)
+    headers = activity_list[0] if activity_list else {}
+    headers = list(headers.keys())
+    markdown = tableToMarkdown(title, activity_list, headers=headers, removeNull=True)
+    return markdown
+
+
+def activity_amp_lookup_to_markdown(results: List[Dict], title: str) -> str:
+    """
+    Parsing the Cisco Umbrella Reporting data
+    Args:
+        results (list): Cisco Umbrella Reporting data
+        title (str): Title string
+    Returns:
+        A string representation of the markdown table
+    """
+    activity_list = []
+    for activity in results:
+        timestamp = activity.get("timestamp", 0)
+        timestamp_string = datetime.utcfromtimestamp(
+            timestamp / 1000.0).strftime(DATE_TIME_FORMAT)
+        new = {
+            "First Seen": activity.get("firstseenat", ''),
+            "Disposition": activity.get("disposition", ''),
+            "Score": activity.get("score", ''),
+            "Host Name": activity.get("hostname", ''),
+            "Malware": activity.get("malwarename", ''),
+            "SHA256": activity.get("sha256", ''),
+            "Date & Time": timestamp_string
+        }
+        activity_list.append(new)
+    headers = activity_list[0] if activity_list else {}
+    headers = list(headers.keys())
+    markdown = tableToMarkdown(title, activity_list, headers=headers, removeNull=True)
+    return markdown
+
+
+def pagination(page: Optional[int], page_size: Optional[int]):
+    """
+    Define pagination.
+    Args:
+        page: The page number.
+        page_size: The number of requested results per page.
+    Returns:
+        limit (int): Records per page.
+        offset (int): The number of records to be skipped.
+    """
+    if (page and page <= 0) or (page_size and page_size <= 0):
+        raise DemistoException(PAGE_NUMBER_ERROR_MSG)
+    page = 0 if not page else page - 1
+    page_size = DEFAULT_PAGE_SIZE if not page_size else page_size
+    limit = page_size
+    offset = page * page_size
+    return limit, offset
+
+
+def get_param(limit: Optional[int], offset: Optional[int], args: Dict) -> Dict:
+    """
+    Setup parameter.
+    Args:
+        limit: Records per page.
+        offset: The number of records to be skipped.
+        args: kwargs
+    Returns:
+        Return arguments dict.
+    """
+    args["limit"] = limit if limit != 50 else args.get('limit', DEFAULT_PAGE_SIZE)
+    args["offset"] = offset
+    args["from"] = args.pop('from', FROM_DATE)
+    args["to"] = args.pop('to', TO_DATE)
```

Comment:
Are the defaults 'from' and 'to' should be -7days -> now for all the commands? 
@meital-aizen - that's what you have decided?

Reply:
Yes, that's ok.